src/entity/builtin_entity.rs

```diff
@@ -671,6 +807,27 @@ impl BuiltinEntityKind {
             BuiltinEntityKind::MusicTrack => &[Language::EN, Language::FR],
         }
     }
+
+    pub fn map_to_supported(&self, language: &Language) -> BuiltinEntityKind {
+        // FIXME: That must not be the right way to get the type of self
+        let builtin_entity_kind = self.clone();
+        let result = match language {
+                // English supports all Datetime subkinds (so all builtin kinds)
+                Language::EN => builtin_entity_kind,
+                // Other language need to remap Datetime subkinds to Datetime
+                _ => {
+                    match builtin_entity_kind {
+                        BuiltinEntityKind::Date |
+                        BuiltinEntityKind::Time |
+                        BuiltinEntityKind::DatePeriod |
+                        BuiltinEntityKind::TimePeriod => BuiltinEntityKind::Datetime,
+                        _ => builtin_entity_kind,
+                    }
+                }
+            };
+        eprintln!("map_to_supported({:?}, {:?}) => {:?}", builtin_entity_kind, language, result);
+        result
+    }
```

Comment:
I'm not sure why we need to remap here. The notion of supporting a builtin entity in a given language is relative to parsing capabilities (is there a parser in english for temperatures?), so it shouldn't affect types and mapping.
Said differently, if a builtin entity is not supported in a certain language, then it won't be output, so no need to remap it to something else.

Reply:
✅ 
Rustling-ontology tagging modified to have the support work like that.